run/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"github.com/paketo-buildpacks/composer"
+	"github.com/paketo-buildpacks/packit/v2"
+	"github.com/paketo-buildpacks/packit/v2/cargo"
+	"github.com/paketo-buildpacks/packit/v2/chronos"
+	"github.com/paketo-buildpacks/packit/v2/draft"
+	"github.com/paketo-buildpacks/packit/v2/postal"
+	"os"
+)
+
+func main() {
+	logEmitter := composer.NewLogEmitter(os.Stdout)
```

Comment:
Nitpick: Would you consider adding debug logging? This is something I've been trying to roll out as we go just to make the developer experience easier. If you want to file an issue to add it later that's cool too. 

If you add it, check out [this PR](https://github.com/paketo-buildpacks/bundler/pull/310/files) where debug logging was added in a similar buildpack.

Reply:
Heads up, I saw you added some logs but for it to take effect in run/main.go youll need to have:
`logEmitter := scribe.NewEmitter(os.Stdout).WithLevel(os.Getenv(BP_LOG_LEVEL))`
and then in one integration test you can set that environment variable and test that the debug logs show up when asserting against the build logs.

Reply:
good catch, thanks @sophiewigmore